.gitignore

```diff
@@ -133,3 +133,4 @@ dmypy.json
 
 # vscode
 .vscode/
+data/chest-xray-pneumonia/chest_xray/__MACOSX/chest_xray/test/NORMAL/._IM-0001-0001.jpeg
```

Comment:
```suggestion
```
I don't think that this line is needed because line 2 already includes an ignore statement for the `data/` directory (as long as it is within `notebooks/`, which should be the case if we run our functions with Jupyter Notebooks!